backends/platform/sdl/win32/win32-main.cpp

```diff
@@ -43,7 +43,7 @@ int __stdcall WinMain(HINSTANCE /*hInst*/, HINSTANCE /*hPrevInst*/,  LPSTR /*lpC
 	SDL_SetModuleHandle(GetModuleHandle(NULL));
 #endif
 // HACK: __argc, __argv are broken and return zero when using mingwrt 4.0+ on MinGW
-// HACK: MinGW-w64 based toolchains neither feature _argc nor _argv. The 32 bit
+//- HACK: MinGW-w64 based toolchains neither feature _argc nor _argv. The 32 bit
```

Comment:
This seems to have been modified by mistake

```suggestion
// HACK: MinGW-w64 based toolchains neither feature _argc nor _argv. The 32 bit
```